File: src/main/java/examples/BusArrivalsExample.java
package examples;

import com.fasterxml.jackson.databind.ObjectMapper;
import model.BusArrivalsResponse;
import service.APIHandler;

public class BusArrivalsExample {

    public static void main(String[] args) {

        APIHandler apiHandler=new APIHandler();

        ObjectMapper objectMapper=new ObjectMapper();

        try {
            BusArrivalsResponse busArrivalsResponse = apiHandler.getBusArrivalsResponse();
            System.out.println("Bus Routes is \n " + objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(busArrivalsResponse));
        } catch(Exception e){
            e.printStackTrace();
            System.out.println("Error Bus Routes is "+e.getMessage());
        }
    }
}

File: src/main/java/examples/BusArrivalsExample.kt
package examples

import kotlin.jvm.JvmStatic
import service.APIHandler
import com.fasterxml.jackson.databind.ObjectMapper
import model.BusArrivalsResponse
import java.lang.Exception

object BusArrivalsExample {
    @JvmStatic
    fun main(args: Array<String>) {
        val apiHandler = APIHandler()
        val objectMapper = ObjectMapper()
        try {
            val busArrivalsResponse = apiHandler.busArrivalsResponse
            println(
                """Bus Routes is 
 ${objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(busArrivalsResponse)}"""
            )
        } catch (e: Exception) {
            e.printStackTrace()
            println("Error Bus Routes is " + e.message)
        }
    }
}